src/config/androidconfig.json

```diff
@@ -1,6 +1,5 @@
 {
-    "supportedRuntimes" : ["android-29", "android-28", "android-27", "android-26", "android-25", "android-24", "android-23"],
-    "supportedBuildTools" : ["29","28","27","26","25","24","23"],
+    "supportedRuntimes" : ["android-30", "android-29", "android-28", "android-27", "android-26", "android-25", "android-24", "android-23"],
```

Comment:
should we change this to something like `minSupportedRuntime` and set it to `android-23` and support version 23+ (so that in the future we wouldn't have to come back here and add `android-31`)?